configure.ac

```diff
@@ -472,10 +472,12 @@ AC_CHECK_SIZEOF(ssize_t,${size_t_size})
 AC_CHECK_SIZEOF(size_t,${size_t_size})
 AC_CHECK_SIZEOF(void*,${voidp_size})
 
+if test "${IS_GR740_MACHINE}" != "yes" ; then
```

Comment:
Doesn't the check_member for mallinfo.arena work on GR740?

Reply:
I believe that the mallinfo.arena  was defined in the Gr740 toolchain but is was empty (stub) or not working properly.